src/services/edusharing/services/EduSharingConnector.js

```diff
@@ -315,7 +335,42 @@ class EduSharingConnector {
 
 			return new EduSharingResponse(parsed);
 		} catch (err) {
-			logger.error('Edu-Sharing failed search ', err.message);
+			logger.error(`Edu-Sharing failed search: ${err.message}`);
+			return Promise.reject(err);
+		}
+	}
+
+	async getPlayerForNode(nodeUuid) {
+		try {
+			const url = `${ES_ENDPOINTS.RENDERER}${nodeUuid}`;
+			const options = {
+				method: 'GET',
+				url,
+				headers: {
+					Accept: 'application/json',
+					...basicAuthorizationHeaders,
+				},
+				timeout: Configuration.get('REQUEST_OPTION__TIMEOUT_ES'),
+			};
+
+			const response = await this.eduSharingRequest(options);
+			const parsed = JSON.parse(response);
+			if (parsed && parsed.detailsSnippet && typeof parsed.detailsSnippet === 'string') {
+				const root = htmlParser.parse(parsed.detailsSnippet);
+				const content = root.querySelector('.edusharing_rendering_content_wrapper');
+				const iframe = content.querySelector('iframe');
+				const iframeSrc = iframe.getAttribute('src');
+				const script = content.querySelector('script');
+				const scriptSrc = script.getAttribute('src');
+				if (iframeSrc === null || scriptSrc === null) {
+					throw new GeneralError('Could not find player in answer from Edu-Sharing');
+				}
+				return { 'iframe_src': iframeSrc, 'script_src': scriptSrc };
+			} else {
+				throw new GeneralError(`Unexpected answer from Edu-Sharing: ${response}`);
```

Comment:
Adding the respone inline is not allowed. Please look to other places how details must be logged. In the way you do it, your by pass our data policy filters in the error pipline and it log more information that in should be.